specifyweb/frontend/js_src/lib/components/Merging/index.tsx

```diff
@@ -208,45 +207,46 @@ function Merging({
            * and, presumably the longest auditing history
            */
           const resources = Array.from(rawResources).sort(
-            sortFunction((resource) => resource.get('timestampCreated'))
+            multiSortFunction(
+              (resource) => resource.get('specifyUser'),
+              true,
+              (resource) => resource.get('timestampCreated')
+            )
           );
           const target = resources[0];
           target.bulkSet(removeKey(merged.toJSON(), 'version'));
 
           const clones = resources.slice(1);
           loading(
-            hijackBackboneAjax(
-              [],
-              async () => target.save(),
-              undefined,
-              'dismissible'
-            ).then(async () => {
-              /*
-               * Make requests sequentially as they are expected to fail
-               * (due to business rules). If we do them sequentially, we
-               * can leave the UI in a state consistent with the back-end
-               */
-              // eslint-disable-next-line functional/no-loop-statement
+            /*
```

Comment:
This comment should no longer be here as we are no longer making requests sequentially

Reply:
Fixed in 2b10695